spec/money_spec.rb

```diff
@@ -241,6 +241,10 @@
     expect((0.33 * Money.new(0.10))).to eq(Money.new(0.03))
   end
 
+  it "handles multiplication for larg numbers" do
+    expect((Money.new("999999999999999999.99") * -1).value).to eql(BigDecimal.new("-999999999999999999.99", 20))
```

Comment:
that's nine hundred ninety-nine quadrillion!!! :) 
https://www.calculatorsoup.com/calculators/conversions/numberstowords.php

Reply:
Yes, this is. And it was in `tax_amout` 💰